### PR TITLE
Repeat bazel build under CI to be sure that it succeeds

### PR DESCRIPTION
**Pull request by cynepco3hahue**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by rmohr** [comment]
Oh we wanted one final run, which would fail before cluster-up

**Comment by cynepco3hahue** [comment]
> Oh we wanted one final run, which would fail before cluster-up

@rmohr We run `make cluster-sync` anyway, so I thought it will be redundant.

**Comment by rmohr** [comment]
Just wanted to fail early.

File: automation/test.sh [hunk 1]
```diff
@@ -155,6 +163,23 @@ if [ -n "${KUBEVIRT_CACHE_FROM}" ]; then
...
+
+# Create .bazelrc to use remote cache
+cat >.bazelrc <<EOF
+startup --host_jvm_args=-Dbazel.DigestFunction=sha256
+build --remote_local_fallback
+build --remote_http_cache=http://bazel-cache.kubevirt-prow.svc.cluster.local:8080/kubevirt.io/kubevirt
+EOF
+
+# build all images with the basic repeat logic
+set +e
```
Comment (rmohr) [inline automation/test.sh]: Can you add a comment here which explains why we do this crazy thing?

Comment (enp0s3) [reply automation/test.sh:178]: https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/7657/pull-kubevirt-e2e-arm64/1520646845313847296

Wiping the greenhouse cache doesn't help. Looking further for a root casue/workaround

**Pull request merged**